desktop/apps/artwork/templates/index.jade

```diff
@@ -7,25 +7,42 @@ block head
   include ../components/meta/index
 
 block body
-  .artwork-page.main-layout-container
-    section.artwork
-      .artwork__main
-        .artwork__main__metadata(
-          class= inPurchaseTestGroup && artwork.is_purchasable ? 'purchase-flow' : ''
-        )
-          include ../components/metadata/index
-          include ../components/commercial/index
-          include ../components/auction/index
-
-        .artwork__main__overview
-          include ../components/banner/index
-          include ../components/images/index
-          include ../components/actions/index
-          include ../components/additional_info/index
-          include ../components/artists/index
-
+  - isAuction = artwork.context.__typename == 'ArtworkContextAuction'
+  
+  if isAuction
+    .artwork-page.main-layout-container
+      section.artwork
+        .artwork__main
+          .artwork__main__overview
+            include ../components/banner/index
+            include ../components/images/index
+            include ../components/actions/index
+            include ../components/additional_info/index
+            include ../components/artists/index
+            
+          include ./metadata
+            
         .artwork__main__overview__fold( class='js-artwork-fold' )
           include ../client/fold
+        
+        .artwork__footer( class='js-artwork-footer' )
+          include ../client/footer
+  
+  else 
```

Comment:
The duplication of code here is meant to indicate that there are separate page layouts. The changes don't seem like much, but inlining them between `if isAuction` leads to more confusion than blocking "pages" off with top level conditionals.